src/components/EDraggable.vue

```diff
@@ -156,6 +203,23 @@ function findCoordinates(el: Element, done: () => void) {
 .EDraggable.isDragging {
   z-index: 100;
 }
+.EDraggable.isVideo .titleBar {
```

Comment:
Not sure EDraggable has to know anything about video. What does it affect, how can it be abstraced?